.github/workflows/Push.yml

```diff
@@ -5,10 +5,11 @@ on: [push, pull_request]
 jobs:
   Test:
     name: Tests
-    runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: ['13.9', '13.10', '13.11']
+        os: ['ubuntu-latest', 'windows-latest']
```

Comment:
I was hoping this was possible, my other option was to just duplicate the entire tests section with Ubuntu and Windows.